proposals/proposal-0001.md

```diff
@@ -0,0 +1,223 @@
+---
+proposal: 0001
+title: "Writing a Proposal"
+authors: [ "Matthew Fisher <matt.fisher@microsoft.com" ]
+created: 2020-08-13
+---
+
+A proposal is a design document providing information to the Helm community, or
+describing a new feature for a Helm project or its processes or environment. The
```

Comment:
In this case, does `Helm project` refer to the Helm client or to all sub-projects of Helm? For example, if Chart Testing is going to do a new major feature should it be through a proposal?